src/profile/utilities.jl

```diff
@@ -0,0 +1,165 @@
+"""
+    TableColumns
+
+A structure containing the column names for the numeric part of the profile table.
+
+The struct also contains a Dict giving the column ranges for Symbols like `:σ` and `:β`.
+Finally it contains a scratch vector used to accumulate to values in a row of the profile table.
+"""
```

Comment:
```suggestion

!!! note
   This is an internal structure used in [`MixedModelProfile`](@ref).
   As such, it may change or disappear in a future release without being considered breaking.

"""
```